test/unit/spec/EnrollChoices_spec.js

```diff
@@ -127,6 +127,36 @@ function (Okta, OktaAuth, Util, EnrollChoicesForm, Beacon, Expect, Router,
       return setup(res);
     }
 
+    function setupMultipleU2FEnrollments(status, enrollment, policyEnrollment) {
+      var res = deepClone(resAllFactors);
+      res.response._embedded.factors[10].status = status;
+      res.response._embedded.factors[10].enrollment = enrollment;
+      res.response._embedded.factors[10].policy.enrollment = policyEnrollment;
+      return setup(res);
+    }
+    
+    function setupMultipleU2FEnrollmentsEnrolledOptional(policyEnrollment) {
+      return setupMultipleU2FEnrollments('ACTIVE', 'OPTIONAL', policyEnrollment);
+    }
+
+    function setupMultipleU2FEnrollmentsRequired(policyEnrollment) {
```

Comment:
following my previous comment, this would become `setupMultipleU2FNotSetupStatusAndRequiredEnrollment `